DEVELOPMENT.md

```diff
@@ -25,6 +25,8 @@ You must install these tools:
 
 1. [`go`](https://golang.org/doc/install): The language `Knative Client` is
    built in (1.11.4 or later)
+1. [`goimports`](https://godoc.org/golang.org/x/tools/cmd/goimports)
```

Comment:
fyi, you don't have to. If not there is a fall back to gofmt and do the import properly on your own. But happy to keep that (as it doesnt harm)

Reply:
I kept getting a warning about goimports not being there, and this seemed to fix it

Reply:
Yep. It's a warning "only". But of course nicer to have that installed, so let's keep that.